macros/schema_tests/_generalized/expression_is_true.sql

```diff
@@ -29,7 +37,7 @@ with grouped_expression as (
         {{ group_by_column }} as col_{{ loop.index }},
         {% endfor -%}
         {% endif %}
-        {{ expression }} as expression
+        {{ dbt_expectations.truth_expression(expression) }}
```

Comment:
Good idea! Might be to add a comment to `truth_expression` why we need that. Without knowing that you're shimming this for another package, readers (i.e. my future self) might be confused.